neo/Trie/MPT/MPTTrie.Put.cs

```diff
@@ -0,0 +1,173 @@
+using Neo.IO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.Trie.MPT
+{
+    public partial class MPTTrie
+    {
+        private ReadOnlySpan<byte> CommonPrefix(ReadOnlySpan<byte> a, ReadOnlySpan<byte> b)
+        {
+            var minLen = a.Length <= b.Length ? a.Length : b.Length;
+            int i = 0;
+            if (a.Length != 0 && b.Length != 0)
```

Comment:
I think that this condition can be removed because the for condition will be 0.

Reply:
done